docs/instrumenten/modelcontractbepalingen.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Modelcontractbepalingen
+toelichting: Modelbepalingen kunnen helpen om een contract op te stellen dat een organisatie in staat stelt veilige en verantwoorde algoritmen of AI-systemen in te kopen.
+levenscyclus:
+- probleemanalyse
+- ontwerp
+- dataverkenning-en-datapreparatie
+- verificatie-en-validatie
+bouwblok:
+- publieke-inkoop
+rollen:
+- aanbestedingsjurist
+- behoeftesteller
+- contractbeheerder
+- inkoopadviseur
+- projectleider
+---
+
+<!-- tags -->
+
+## Instrument
+
+Modelbepalingen kunnen helpen om een contract op te stellen dat een organisatie in staat stelt veilige en verantwoorde algoritmen of AI-systemen in te kopen. Deze bepalingen (of voorwaarden) kunnen opgenomen worden wanneer er een contract wordt afgesloten met een leverancier van een algoritme of algoritmisch systeem. Er kunnen dan bijvoorbeeld beperkingen gelden om onaanvaardbare risico's van AI te vermijden, of bepaalde voorwaarden gesteld worden waaraan een algoritme juist moet voldoen. Ook kunnen bepaalde [voorwaarden worden opgenomen op basis van de vereisten](../maatregelen/maak_vereiste_onder_van_contractvoorwaarden.md) in het Algoritmekader.
+
+### Europese Commissie
+De Europese contractvoorwaarden voor AI bieden aanbestedende organisaties de mogelijkheid om specifieke clausules op te nemen in de overeenkomst. Op deze manier worden afspraken gemaakt over onderwerpen die in lijn zijn met de aankomende AI-Act. Er zijn 2 versies van de AI-inkoopvoorwaarden opgesteld: een set voorwaarden voor AI-toepassingen met een hoog-risicoprofiel en een set voorwaarden voor AI-toepassingen met een laag-risicoprofiel.
```

Comment:
Hier verwijzen naar de beide inkoopvoorwaarden?

Reply:
Ik dacht eigenlijk: misschien kunnen we een URL toevoegen naar de inkoopvoorwaarden? Nu moet je als gebruiker toch nog zelf gaan zoeken?